### DISCUSSION TITLE
Better xCloud works with Violentmonkey.

### DISCUSSION THREAD

**Discussion by ww1h** [redphx/better-xcloud#478] [General]
Was surprised to learn tampermonkey is closed source. I tried Better xCloud with violentmonkey and I can happily report it works well, at least for the settings I use:
✔ Preferred game's language
✔ Target resolution
✔ Visual quality
✔ Maximum video bitrate
✔ Disable post stream feedback dialog
✔ Reduce UI animations
✔ Disable social features
✔ Disable xCloud analytics

**Comment by redphx** [comment]
It should work as long as it support `document-start` flag. But unlike Tampermonkey, Violentmonkey doesn't Manifest V3 yet https://github.com/violentmonkey/violentmonkey/issues/1934
